.github/stale.yml

```diff
@@ -0,0 +1,42 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 14
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - Merge After Thaw
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This pull request has been automatically marked as stale because it has not
+  had activity in 14 days.
+  If you think this pull request should be merged, please check
+
+  - that all unit tests are passing
+
+  - that all comments by reviewers have been addressed
+
+  - that there is enough information for reviewers, in particular
+
+    - link to any issues which this pull request fixes
+
+    - add a description of workflows which this pull request fixes
+
+    - add screenshots if applicable
+
+  - that you have written unit tests where possible
+
+  In case you should have any uncertainty, please leave a comment and we will
+  happy to help you proceed with this pull request.
```

Comment:
_be_ happy

Reply:
Thanks  @DelazJ , always good to have your eagle eyes on those texts